src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
@videlalvaro currently we block all publishers. I can understand this for RAM alarms because anything that enters a node consumes RAM. For disk alarms this sounds like an overkill and we still cannot guarantee the node won't run out of disk.